package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.11.0",
   "description": "Common Brightspace eslint configs.",
   "main": "index.js",
+  "publishConfig": {
```

Comment:
the Semantic release plugin specified that this needed to be present for the other orgs.  I assumed that it was relevant here as well

Reply:
Ah, this is because by default, NPM publishes scoped packages (i.e. `@someorg/...`) as private.  The `@brightspace-ui` or `@brightspace-ui-labs` orgs do not support private packages (not sure about any other orgs we have) so this config is necessary to publish to them, otherwise the publish would fail. In this particular case, the package isn't scoped, so this config should not be necessary, but I don't think it hurts either.

Reply:
> Ah, this is because by default, NPM publishes scoped packages (i.e. @someorg/...) as private. The @brightspace-ui or @brightspace-ui-labs orgs do not support private packages (not sure about any other orgs we have) so this config is necessary to publish to them, otherwise the publish would fail. In this particular case, the package isn't scoped, so this config should not be necessary, but I don't think it hurts either.

TIL!